Formula/libbi.rb

```diff
@@ -0,0 +1,156 @@
+class Libbi < Formula
+  desc "Bayesian state-space modelling on parallel computer hardware"
+  homepage "http://libbi.org"
+  head "https://github.com/libbi/LibBi.git"
+
+  stable do
+    url "https://github.com/libbi/LibBi/archive/1.3.0.tar.gz"
+    sha256 "0dd313dd71e72b2f16ca9074800fc2fa8bf585bec3b87a750ff27e467a9826d0"
+  end
+
+  option "without-test", "Disable build-time checking (not recommended)"
+
+  depends_on "perl"
```

Comment:
Does it not build with system perl?

Reply:
It does, but if someone were to install a brewed perl later this will break libbi as the dependencies will have been compiled with a different version. In the past, I've found that forcing it to use a brewed perl is the safest option to prevent conflicts (hence also the setting of PATH further down to ensure the brewed perl is used), but I'm happy to remove these lines (or replace with `depends_on :perl`?) if that's preferred.

Reply:
By far, most users don't have perl installed. So let's make it work for them with system perl. Remove the dependency (and `:perl` is being phased out), and the PATH hack below.